f/folks/manifest.x86_64.jsonc

```diff
@@ -0,0 +1,148 @@
+/** Human readable report. This is not consumed by boulder */
+{
+	"manifest-version": "0.2",
+	"packages": {
+		"folks": {
+			"build-depends": [
+				"binary(g-ir-compiler)",
+				"binary(vala)",
+				"pkgconfig(dbus-glib-1)",
+				"pkgconfig(evolution-data-server-1.2)",
+				"pkgconfig(gee-0.8)",
+				"pkgconfig(gio-2.0)",
+				"pkgconfig(glib-2.0)",
+				"pkgconfig(gmodule-2.0)",
+				"pkgconfig(gobject-2.0)",
+				"pkgconfig(libebook-1.2)",
+				"pkgconfig(libebook-contacts-1.2)",
+				"pkgconfig(libedataserver-1.2)",
+				"pkgconfig(libxml-2.0)",
+				"pkgconfig(readline)",
+				"pkgconfig(sysprof-capture-4)",
+				"pkgconfig(telepathy-glib)",
+				"python-dbusmock"
+			],
+			"depends": [
+				"interpreter(/usr/lib/ld-linux-x86-64.so.2(x86_64))",
+				"soname(libc.so.6(x86_64))",
+				"soname(libebook-1.2.so.21(x86_64))",
+				"soname(libebook-contacts-1.2.so.4(x86_64))",
+				"soname(libedataserver-1.2.so.27(x86_64))",
+				"soname(libgee-0.8.so.2(x86_64))",
+				"soname(libgio-2.0.so.0(x86_64))",
+				"soname(libglib-2.0.so.0(x86_64))",
+				"soname(libgmodule-2.0.so.0(x86_64))",
+				"soname(libgobject-2.0.so.0(x86_64))",
+				"soname(libm.so.6(x86_64))",
+				"soname(libreadline.so.8(x86_64))",
+				"soname(libtelepathy-glib.so.0(x86_64))",
+				"soname(libxml2.so.16(x86_64))"
+			],
+			"files": [
+				"/usr/bin/folks-import",
+				"/usr/bin/folks-inspect",
+				"/usr/lib/folks/26/backends/bluez/bluez.so",
+				"/usr/lib/folks/26/backends/dummy/dummy.so",
+				"/usr/lib/folks/26/backends/eds/eds.so",
+				"/usr/lib/folks/26/backends/key-file/key-file.so",
+				"/usr/lib/folks/26/backends/ofono/ofono.so",
+				"/usr/lib/folks/26/backends/telepathy/telepathy.so",
```

Comment:
I think the telepathy backend and tools can be split out to subpackages like how Fedora does so.